membership/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.conf import settings
+# Create your models here.
+
+MEMBERSHIP_CHOICES = (
+('Premium', 'pre'),
+('Free', 'free')
```

Comment:
I think these tuple values are backwards. It should be `('db-value', 'Display Value')`